### DISCUSSION TITLE
What is the *-audit.json?

### DISCUSSION THREAD

**Discussion by electrofloat** [zwave-js/zwave-js#4329] [General] [answered]
I'm running this server manually (meaning /home/zwavejs/zwave-js-server/node_modules/.bin/zwave-server /dev/tty0 --config /home/zwavejs/zwavejs-config.js") configured to log into a file.

I can see that a *-audit.log file is created in the directory where the actual log file is. In the json there are some "settings" it seems.
What are these? What are the possible values of them?

does keep, days=true, amount=7 means that it will create a new log file every day, and it will keep the last 7 of them?

Why is it so hard to find documentation about this?

**Comment by AlCalzone** [comment, marked as answer]
That's from https://github.com/winstonjs/winston-daily-rotate-file, which we have configured to rotate the log files over 7 days.

**Reply by electrofloat** [reply]
I see. But can I as a user of the zwave-server configure this the way I want to, or is that "burned in" and cannot be changed?

**Reply by AlCalzone** [reply]
You could open an issue with a feature request to add options for this. Currently there's no way to change it.
